pkgs/by-name/pa/parsify/package.nix

```
@@ -0,0 +1,26 @@
+{ lib,
+appimageTools,
+fetchurl
+}:
```

nitpicky style comment, but someone else'll complain about it even if I don't:

```suggestion
{ lib
, appimageTools
, fetchurl
}:
```

---

fixed in https://github.com/NixOS/nixpkgs/pull/274845/commits/82038709e94c983608708239f2b8ec82ed8b2532